esm/pretrained.py

```diff
@@ -304,3 +328,17 @@ def esm_if1_gvp4_t16_142M_UR50():
     Returns a tuple of (Model, Alphabet).
     """
     return load_model_and_alphabet_hub("esm_if1_gvp4_t16_142M_UR50")
+
+
+def esm_if_gvpgnnlarge_gvp16_21M_UR50():
+    """Inverse folding model with 21M params, with 8 GVP-GNN encoder layers and
+    8 GVP-GNN decoder layers, trained on CATH structures and 12 million
+    alphafold2 predicted structures from UniRef50 sequences.
+
+    Returns a tuple of (Model, Alphabet).
+    """
+    model_path = "/home/chloehsu/inverse_opensource/esm_public_fork/esm/tests/esm_if_gvp_gnn_model.pt"
```

Comment:
Now uploaded to `s3://dl.fbaipublicfiles.com/fair-esm/models/esm_if1_gvpgnnlarge_gvp16_21M_UR50.pt`